happypanda/database/db_constants.py

```diff
@@ -14,22 +14,30 @@
 
 import os
 
+from appdirs import user_data_dir
+
+from happypanda.database.arguments import args
+
+appauthor = "Pewpews"
```

Comment:
This is the exact variable appdirs uses but sure, I suppose I can just specify it when calling user_data_dir like: `user_data_dir(appauthor=__author__, etc, etc)`